python/lsst/jointcal/jointcal.py

```diff
@@ -135,7 +156,8 @@ def _build_ccdImage(self, dataRef, associations, jointcalControl):
 
         goodSrc = self.sourceSelector.selectSources(src)
 
-        # TODO: NOTE: Leaving the old catalog and comparison code in while we
+        # ------------------
+        # TODO: Leaving the old catalog and comparison code in while we
```

Comment:
Code should not be left commented out.  I know you have philosophical disagreements, but I think you should submit to the way things are done until you have explicit approval to change them.